### ISSUE TITLE
Can we add @stripe/stripe-react-native package for React native web version will it support?

### ISSUE THREAD

**Issue by GitHubShasiKumar** [stripe/stripe-react-native#1556]

**Comment by majirosstefan** [comment]
No. At least "@stripe/stripe-react-native": "0.6.1", with  "react-native": "0.68.2" and   "react-native-web": "0.17.7" is causing error:  "Cannot read properties of undefined (reading 'setGlobalHandler'

**Comment by alejomendoza** [comment]
any ideas if there are plans to support web? what is missing for it to work?

**Comment by majirosstefan** [comment]
The workaround (solution?) that I saw was add to Stripe for React JS (web SDK) to your React Native app and then write your own platform specific utilities / wrappers e.g. in a files called

- `stripe.tsx`  - for iOS, android, etc that uses imports from React Native SDK
- `stripe.web.tsx` for web that uses imports from Stripe ReactJS (web SDK)

The bundler would pick the correct file for your platform. However, the both files need to implement the same interface or you would need to use branching in your code (e.g. `if (Platform.OS == "web") {} else {}`)

**Comment by naticio** [comment]
> The workaround (solution?) that I saw was add to Stripe for React JS (web SDK) to your React Native app and then write your own platform specific utilities / wrappers e.g. in a files called
> 
> * `stripe.tsx`  - for iOS, android, etc that uses imports from React Native SDK
> * `stripe.web.tsx` for web that uses imports from Stripe ReactJS (web SDK)
> 
> The bundler would pick the correct file for your platform. However, the both files need to implement the same interface or you would need to use branching in your code (e.g. `if (Platform.OS == "web") {} else {}`)

does this work for expo apps?

**Comment by moza88** [comment]
Is there an example of this?

**Comment by sfriedman-stripe** [comment]
Hi all, we do not plan on supporting the web with this SDK. Please use Stripe.js
